### DISCUSSION TITLE
Account suspended

### DISCUSSION THREAD

**Discussion by redev007** [community/community#73734] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi ,
 Please my friend have Account [@jyotidwi](https://github.com/jyotidwi)  wanted to login into his github account and he saw a message that his account was suspended without any reason 😔 and  he have been waiting last 7 days .  How could he get his account back because all his documents and projects are there and  now not access by anyone. 
        
username  : Jyotidwi 
Email : jyotsnanand.jnd2000@gmail.com                                                                                                              

 1 Ticket ID:2412014                                                                                                                                              2 Ticket ID:2421065 

![Screenshot_20231102-225723_Chrome](https://github.com/community/community/assets/109581566/bad31ee3-9aa3-45c4-a1a4-b58afcf14a62)

**Comment by doducduy20127477** [comment, marked as answer]
### Your friend may follow these steps:

1. **Open a Support Ticket:** Your friend has already taken the first step by opening support tickets (Ticket ID: 2412014 and Ticket ID: 2421065). It's essential to provide as much detail and context as possible in these tickets. Be patient, as it may take some time for GitHub's support team to investigate and respond.
2. **Check Email:** Encourage your friend to check their email regularly, including spam or junk folders. GitHub's support team may send communication regarding the suspension or steps to follow for reinstatement via email.
3. Follow GitHub's Instructions: If GitHub's support team provides instructions on how to address the issue, your friend should follow them carefully. This may involve providing additional information or taking specific actions.
4. **Appeal Respectfully:** If your friend believes the suspension was a mistake or there was a misunderstanding, they should appeal the suspension in a respectful and clear manner through the support tickets. Explain the situation and the importance of regaining access to the account.
5. **Maintain Patience:** Dealing with account suspensions can be frustrating, but it's important to remain patient and allow GitHub's support team sufficient time to investigate and respond to the situation.
6. **Verify Account Ownership:** Your friend may be asked to verify their identity to prove ownership of the account. Be prepared to provide any necessary verification documents or information.
7. Engage with GitHub's Community: GitHub has an active community forum where similar issues are discussed. Your friend can seek advice from others who may have experienced similar situations.
8. **Use the Appeals Process:** If the initial support tickets do not yield a satisfactory resolution, your friend can explore GitHub's appeals process for account suspensions. This process may involve reaching out to GitHub's support team again.

_If my answer help you resolve problems, please "Mark as answer" it._

**Reply by ghost** [reply]
Hi,

My GitHub account: https://github.com/hv33y was suspended. My ticket: [2867879].

I don't know why as I didn't receive any mails or hints indicating why this happened.
I was taking to some one here in the community trying to solve a problem when suddenly it logged me out and when tried to sign in a message appear that account suspended for violating GitHub’s terms of service. Can some one please what is happening?
I am so sad and can't describe my sadness. I never violated TOS in my life and if that happened it is absolutely a mistake and not intentional. I’m happy to oblige with all of GitHub’s policies and Terms.

Thank you!

**Reply by rahulgotrekiya-2** [reply]
i have same problem :(

**Reply by MadaraNiseron** [reply]
now i see your account it recovered, how long it takes to get your account back bro ?

**Reply by rahulgotrekiya** [reply]
After raising the token, they replied, and 3–4 days later, my account was back.

**Reply by MadaraNiseron** [reply]
ok thank you for information brother

**Comment by Mee-meet** [comment]
Subject: Request for Unsuspension of GitHub Account

Dear GitHub Support Team,

I hope this message finds you well. My name is Meet Mehta, and I am writing to request the unsuspension of my GitHub account. The details of my account are as follows:

Username: MeetInCode
Email: meets.studentlife@gmail.com
I recently discovered that my account has been suspended, and I am unsure of the reason for this action. I rely heavily on GitHub for my studies and projects, and losing access to my repositories has caused significant disruption.

If the suspension was due to a misunderstanding or an error on my part, I sincerely apologize and am willing to take any necessary corrective actions to ensure compliance with GitHub's terms of service.

I would greatly appreciate your assistance in reviewing my case and providing guidance on how to resolve this issue. Thank you for your understanding and support.

Looking forward to your prompt response.

Best regards,
Meet Mehta

**Comment by shinybrightstar** [comment]
Hi @rahulgotrekiya-2 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

**Comment by siya0in** [comment]
@github-staff

My GitHub account (ssing645, associated with ssing645@asu.edu
) has been unexpectedly suspended. I’m unsure of the reason, as I’ve always followed GitHub’s usage and community guidelines.

This account is critical for my academic and work projects — I actively maintain repositories for ongoing university and professional work. 
I kindly request that my account be reviewed and reinstated as soon as possible. Please let me know if you need any verification or further details from my side.

Thank you for your time and understanding.
Name: Siya Singh
GitHub Username: ssing645
Email: ssing645@asu.edu

**Comment by nahid789898** [comment]
Hello GitHub Support Team,

My account (@Hackernahid) has been suspended recently, and I would like to appeal for a review.  
I was developing an open-source AI chatbot project called “Airi 2.0”, designed for Termux users — it is completely educational and does not violate any terms of service.  

If any file (like config.json or API key) was mistakenly uploaded, I have already removed it and ensured that such sensitive data will never be committed again.  
I deeply respect GitHub’s community guidelines and apologize for any misunderstanding.

Please review my account and consider restoring access.  
I value GitHub as a platform for learning, collaboration, and open-source contribution.

Thank you for your time and support. 🙏

Best regards,  
Nahid Hasan (@Hackernahid)
Bangladesh Cyber Panthers (BCP)

**Comment by Harsh28072002** [comment]
<img width="1901" height="947" alt="suspended " src="https://github.com/user-attachments/assets/568a1fd6-1314-41e5-aa60-346e848c0e8c" />


Hello GitHub Support Team,

My account  (Harsh072002-cmd) was suspended for an alleged Terms of Service violation. I believe this may be an error. I was using GitHub for personal learning and DevOps practice, specifically Terraform, Microsoft Azure, and GitHub Actions CI/CD. I was testing a Terraform CI workflow and pushing normal infrastructure-as-code commits to my repository. I did not intentionally engage in spam, abuse, unauthorized access, or any activity intended to violate GitHub's policies. I would appreciate a review of the suspension and clarification of the activity that caused the restriction.

I request your team to please review my account and please restore it.

Thanks and Regards,
Harsh Vardhan
